firefox_ui_tests/firefox_ui_tests/functional/security/test_safe_browsing_primary_download.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sys
+
+from marionette_driver import By, Wait
+from firefox_puppeteer.testcases import FirefoxTestCase
+
+
+class TestSafeBrowsingPrimaryDownload(FirefoxTestCase):
+
+    data = [
+        {
+            'platform_list': ['linux2', 'win32', 'darwin'],
```

Comment:
As suggested please use `platforms` here so it is equal to `files`. Usually you should not use the data type in the variable name given that it will cause problems when the type changes.